presets/4.3/tune/Tehllama_533_TinyTrainer_3in_3S_TUNE_Options.txt

```diff
@@ -0,0 +1,156 @@
+#$ TITLE: FlyFive33 3in TinyTrainer 3S Tune
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: TinyTrainer, 533, Spec, Race, Llama
+#$ AUTHOR: Daniel Appel / Tehllama && Armando Gallegos / Mondo && Evan Turner / HeadsUp
+#$ DESCRIPTION: This is an adaptation of the 533 TinyTrainer tune, and expects bidirectional DShot ESCs at 48kHz PWM with MedHigh timing. 
+#$ DESCRIPTION: These fly best on 3S batteries in the 450-550mAh capacity range with 533 1404 motors.
+#$ DISCLAIMER: This will install the tune onto Profile #3, and auto-select when a 3S battery is plugged in.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/233
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+# name: FIVE33
+
+# master
+
+set dshot_idle_value = 700
+# Fallback if Dynamic Idle not working or not selected
+set dshot_bidir = ON
+
+set simplified_gyro_filter_multiplier = 130
+
+
+
+profile 2
+# profile 2
+set auto_profile_cell_count = 3
```

Comment:
not allowed in TUNE category. Either change category to OTHER or remove

Reply:
Removed